Jenkins build log indicates Java codes contains non ASCII characters

Jenkins build log indicates Java codes contains non ASCII characters

```
:libraries:couchbase-lite-java-core:compileJavawarning: [options] bootstrap class path not set in conjunction with -source 1.6
/home/couchbase/jenkins/workspace/couchbase-lite-android-build/libraries/couchbase-lite-java-core/src/main/java/com/couchbase/lite/replicator/PullerInternal.java:159: error: unmappable character for encoding ASCII
            // array of revisions in ???inbox??? and removes the ones that already exist. So whatever???s left in ???inbox???
                                     ^
/home/couchbase/jenkins/workspace/couchbase-lite-android-build/libraries/couchbase-lite-java-core/src/main/java/com/couchbase/lite/replicator/PullerInternal.java:159: error: unmappable character for encoding ASCII
            // array of revisions in ???inbox??? and removes the ones that already exist. So whatever???s left in ???inbox???
                                      ^
/home/couchbase/jenkins/workspace/couchbase-lite-android-build/libraries/couchbase-lite-java-core/src/main/java/com/couchbase/lite/replicator/PullerInternal.java:159: error: unmappable character for encoding ASCII
            // array of revisions in ???inbox??? and removes the ones that already exist. So whatever???s left in ???inbox???
                                       ^
/home/couchbase/jenkins/workspace/couchbase-lite-android-build/libraries/couchbase-lite-java-core/src/main/java/com/couchbase/lite/replicator/PullerInternal.java:159: error: unmappable character for encoding ASCII
            // array of revisions in ???inbox??? and removes the ones that already exist. So whatever???s left in ???inbox???
                                             ^
/home/couchbase/jenkins/workspace/couchbase-lite-android-build/libraries/couchbase-lite-java-core/src/main/java/com/couchbase/lite/replicator/PullerInternal.java:159: error: unmappable character for encoding ASCII
            // array of revisions in ???inbox??? and removes the ones that already exist. So whatever???s left in ???inbox???
                                              ^
/home/couchbase/jenkins/workspace/couchbase-lite-android-build/libraries/couchbase-lite-java-core/src/main/java/com/couchbase/lite/replicator/PullerInternal.java:159: error: unmappable character for encoding ASCII
            // array of revisions in ???inbox??? and removes the ones that already exist. So whatever???s left in ???inbox???
                                               ^
/home/couchbase/jenkins/workspace/couchbase-lite-android-build/libraries/couchbase-lite-java-core/src/main/java/com/couchbase/lite/replicator/PullerInternal.java:159: error: unmappable character for encoding ASCII
            // array of revisions in ???inbox??? and removes the ones that already exist. So whatever???s left in ???inbox???
                                                                                                     ^
/home/couchbase/jenkins/workspace/couchbase-lite-android-build/libraries/couchbase-lite-java-core/src/main/java/com/couchbase/lite/replicator/PullerInternal.java:159: error: unmappable character for encoding ASCII
            // array of revisions in ???inbox??? and removes the ones that already exist. So whatever???s left in ???inbox???
                                                                                                      ^
/home/couchbase/jenkins/workspace/couchbase-lite-android-build/libraries/couchbase-lite-java-core/src/main/java/com/couchbase/lite/replicator/PullerInternal.java:159: error: unmappable character for encoding ASCII
            // array of revisions in ???inbox??? and removes the ones that already exist. So whatever???s left in ???inbox???
                                                                                                       ^
/home/couchbase/jenkins/workspace/couchbase-lite-android-build/libraries/couchbase-lite-java-core/src/main/java/com/couchbase/lite/replicator/PullerInternal.java:159: error: unmappable character for encoding ASCII
            // array of revisions in ???inbox??? and removes the ones that already exist. So whatever???s left in ???inbox???
                                                                                                                  ^
/home/couchbase/jenkins/workspace/couchbase-lite-android-build/libraries/couchbase-lite-java-core/src/main/java/com/couchbase/lite/replicator/PullerInternal.java:159: error: unmappable character for encoding ASCII
            // array of revisions in ???inbox??? and removes the ones that already exist. So whatever???s left in ???inbox???
                                                                                                                   ^
/home/couchbase/jenkins/workspace/couchbase-lite-android-build/libraries/couchbase-lite-java-core/src/main/java/com/couchbase/lite/replicator/PullerInternal.java:159: error: unmappable character for encoding ASCII
            // array of revisions in ???inbox??? and removes the ones that already exist. So whatever???s left in ???inbox???
                                                                                                                    ^
/home/couchbase/jenkins/workspace/couchbase-lite-android-build/libraries/couchbase-lite-java-core/src/main/java/com/couchbase/lite/replicator/PullerInternal.java:159: error: unmappable character for encoding ASCII
            // array of revisions in ???inbox??? and removes the ones that already exist. So whatever???s left in ???inbox???
                                                                                                                          ^
/home/couchbase/jenkins/workspace/couchbase-lite-android-build/libraries/couchbase-lite-java-core/src/main/java/com/couchbase/lite/replicator/PullerInternal.java:159: error: unmappable character for encoding ASCII
            // array of revisions in ???inbox??? and removes the ones that already exist. So whatever???s left in ???inbox???
                                                                                                                           ^
/home/couchbase/jenkins/workspace/couchbase-lite-android-build/libraries/couchbase-lite-java-core/src/main/java/com/couchbase/lite/replicator/PullerInternal.java:159: error: unmappable character for encoding ASCII
            // array of revisions in ???inbox??? and removes the ones that already exist. So whatever???s left in ???inbox???
                                                                                                                            ^
/home/couchbase/jenkins/workspace/couchbase-lite-android-build/libraries/couchbase-lite-java-core/src/main/java/com/couchbase/lite/replicator/PullerInternal.java:175: error: unmappable character for encoding ASCII
            // May decrease the changesCount, to account for the revisions we just found out we don???t need to get.
                                                                                                   ^
/home/couchbase/jenkins/workspace/couchbase-lite-android-build/libraries/couchbase-lite-java-core/src/main/java/com/couchbase/lite/replicator/PullerInternal.java:175: error: unmappable character for encoding ASCII
            // May decrease the changesCount, to account for the revisions we just found out we don???t need to get.
                                                                                                    ^
/home/couchbase/jenkins/workspace/couchbase-lite-android-build/libraries/couchbase-lite-java-core/src/main/java/com/couchbase/lite/replicator/PullerInternal.java:175: error: unmappable character for encoding ASCII
            // May decrease the changesCount, to account for the revisions we just found out we don???t need to get.
                                                                                                     ^
/home/couchbase/jenkins/workspace/couchbase-lite-android-build/libraries/couchbase-lite-java-core/src/main/java/com/couchbase/lite/internal/InterfaceAudience.java:6: error: unmappable character for encoding ASCII
 * help with the issue that Java???s scoping is not very complete. One is often forced to
                                ^
/home/couchbase/jenkins/workspace/couchbase-lite-android-build/libraries/couchbase-lite-java-core/src/main/java/com/couchbase/lite/internal/InterfaceAudience.java:6: error: unmappable character for encoding ASCII
 * help with the issue that Java???s scoping is not very complete. One is often forced to
                                 ^
/home/couchbase/jenkins/workspace/couchbase-lite-android-build/libraries/couchbase-lite-java-core/src/main/java/com/couchbase/lite/internal/InterfaceAudience.java:6: error: unmappable character for encoding ASCII
 * help with the issue that Java???s scoping is not very complete. One is often forced to
```

--- a/src/main/java/com/couchbase/lite/internal/InterfaceAudience.java
+++ b/src/main/java/com/couchbase/lite/internal/InterfaceAudience.java
@@ -3,7 +3,7 @@ package com.couchbase.lite.internal;
 
 /**
  * Annotations to help mark methods as being public or private.  This is needed to
- * help with the issue that Java’s scoping is not very complete. One is often forced to
+ * help with the issue that Java's scoping is not very complete. One is often forced to
  * make a class public in order for other internal components to use it. It does not have
  * friends or sub-package-private like C++
  *

--- a/src/main/java/com/couchbase/lite/replicator/PullerInternal.java
+++ b/src/main/java/com/couchbase/lite/replicator/PullerInternal.java
@@ -156,7 +156,7 @@ public class PullerInternal extends ReplicationInternal implements ChangeTracker
         int numRevisionsRemoved = 0;
         try {
             // findMissingRevisions is the local equivalent of _revs_diff. it looks at the
-            // array of revisions in ‘inbox’ and removes the ones that already exist. So whatever’s left in ‘inbox’
+            // array of revisions in "inbox" and removes the ones that already exist. So whatever’s left in ‘inbox’
             // afterwards are the revisions that need to be downloaded.
             numRevisionsRemoved = db.findMissingRevisions(inbox);
         } catch (SQLException e) {
@@ -172,7 +172,7 @@ public class PullerInternal extends ReplicationInternal implements ChangeTracker
 
         if (numRevisionsRemoved > 0) {
             Log.v(Log.TAG_SYNC, "%s: processInbox() setting changesCount to: %s", this, getChangesCount().get() - numRevisionsRemoved);
-            // May decrease the changesCount, to account for the revisions we just found out we don’t need to get.
+            // May decrease the changesCount, to account for the revisions we just found out we don't need to get.
             addToChangesCount(-1 * numRevisionsRemoved);
         }
 

